packages/common/src/directives/ng_optimized_image/lcp_image_observer.ts

```diff
@@ -15,6 +15,13 @@ import {assertDevMode} from './asserts';
 import {imgDirectiveDetails} from './error_helper';
 import {getUrl} from './url';
 
+interface ObservedImage {
```

Comment:
nit: I think adding "state" would make it more clear what this type represents.
```suggestion
interface ObservedImageState {
```

Reply:
Sounds good--done.